backend/open_webui/models/chats.py

```
@@ -100,67 +169,248 @@ class ChatTitleIdResponse(BaseModel):
 
-            result = Chat(**chat.model_dump())
-            db.add(result)
+            db.add_all(chats)
             db.commit()
-            db.refresh(result)
-            return ChatModel.model_validate(result) if result else None
 
-    def update_chat_by_id(self, id: str, chat: dict) -> Optional[ChatModel]:
+            # Dual-write messages to chat_message table
+            try:
+                for form_data, chat_obj in zip(chat_import_forms, chats):
```

Ruff (0.15.4)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)